interface/weno/adminClass.php

```diff
@@ -0,0 +1,736 @@
+                        `Mail_City` varchar(10) DEFAULT NULL,
+                        `Mail_State` varchar(2) DEFAULT NULL,
+                        `Mail_Zip_Code` int(9) DEFAULT NULL,
+                        `Mail_Phone` int(10) DEFAULT NULL,
+                        `Mail_Fax` int(10) DEFAULT NULL,
+                        `EPCS_Permitted` varchar(5) DEFAULT NULL,
+                        `Accept_NewRx` varchar(4) DEFAULT NULL,
+                        `Accept_Refillresponse` varchar(5) DEFAULT NULL,
+                        `Accept_RxChangeresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Verify` varchar(5) DEFAULT NULL,
+                        `Accept_Cancelrx` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryrequest` varchar(5) DEFAULT NULL,
+                        `Accept_RxHistoryresponse` varchar(5) DEFAULT NULL,
+                        `Accept_Census` varchar(5) DEFAULT NULL,
+                        `Accept_Resupply` varchar(5) DEFAULT NULL,
+                        PRIMARY KEY (`id`)
+                        ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=11150"
+    		);
+
+          sqlStatement(
+            "CREATE TABLE IF NOT EXISTS `erx_rx_log` (
+                         `id` int(20) NOT NULL AUTO_INCREMENT,
+                         `prescription_id` int(6) NOT NULL,
+                         `date` varchar(25) NOT NULL,
+                         `time` varchar(15) NOT NULL,
+                         `code` int(6) NOT NULL,
+                         `status` text NOT NULL,
+                         `message_id` varchar(100) DEFAULT NULL,
+                         `read` int(1) DEFAULT NULL,
+                         PRIMARY KEY (`id`)
+                          ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=133" 
+          	);
+
+          sqlStatement(
+              "CREATE TABLE IF NOT EXISTS `erx_erx_narcotics` (
+                            `id` int(11) NOT NULL AUTO_INCREMENT,
+                            `drug` varchar(255) NOT NULL,
+                            `dea_number` varchar(5) NOT NULL,
+                            `csa_sch` varchar(2) NOT NULL,
+                            `narc` varchar(2) NOT NULL,
+                            `other_names` varchar(255) NOT NULL,
+                             PRIMARY KEY (`id`)
+                           ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 AUTO_INCREMENT=570"
+          	);
+
+
+/*          
+          $lastPharmacy = sqlQuery("SELECT id FROM `pharmacies` ORDER BY id DESC LIMIT 1 ");
+           $sqlPharmacy = "INSERT INTO `pharmacies` (`id`, `name`, `transmit_method`, `email`, `ncpdp`, `npi`) VALUES
+                           (?, 'Test Direct Pharmacy', 1, NULL, 12345678, 321),
+                           (?, 'CCS Medical', 1, NULL, 1086203, 0)";
+            $addPharmacy = array($lastPharmacy+1, $lastPharmacy+2);
+           sqlStatement($sqlPharmacy, $addPharmacy);
+*/
+
+          $sqlNarc =
```

Comment:
Ok, that helps. This data will rarely change. I have been working with an anesthesiologist for the past 5 yrs and the list of narcotics that are not allowed to be electronically prescribed has not changed. However, this list was downloaded from the DEA site via PDF that was scrapped into this. The DEA only puts this list out in a PDF. So I have to manually check the site each year to see if anything has changed and in 5 years. It has not. Not to say it won't but change comes slowly. 
However, I am still open to a better way to do this.